extension/src/popup/components/accountHistory/HistoryItem/index.tsx

```diff
@@ -256,7 +288,7 @@ export const HistoryItem = ({
 
               const formattedTokenAmount = formatTokenAmount(
                 new BigNumber(attrs.amount),
-                _token.decimals,
+                _token.decimals!,
```

Comment:
This should be ok because if any of the rpc calls to get token details fail, we will hit the catch block which renders a more generic history item row without token details.